perception/traffic_light_classifier/launch/traffic_light_classifier.launch.xml

```diff
@@ -2,32 +2,12 @@
   <arg name="input/image" default="~/image_raw"/>
   <arg name="input/rois" default="~/rois"/>
   <arg name="output/traffic_signals" default="classified/traffic_signals"/>
-  <arg name="data_path" default="$(env HOME)/autoware_data" description="packages data and artifacts directory path"/>
-  <arg name="classifier_label_path" default="$(var data_path)/traffic_light_classifier/lamp_labels.txt" description="classifier label path"/>
-  <arg name="classifier_model_path" default="$(var data_path)/traffic_light_classifier/traffic_light_classifier_mobilenetv2_batch_6.onnx" description="classifier onnx model path"/>
-  <arg name="classifier_precision" default="fp16"/>
-
-  <arg name="use_gpu" default="true"/>
-  <!-- classifier_type {hsv_filter: 0, cnn: 1} -->
-  <arg name="classifier_type" default="1" if="$(var use_gpu)"/>
-  <arg name="classifier_type" default="0" unless="$(var use_gpu)"/>
-  <!-- classify_traffic_light_type {car: 0, pedestrian:1} -->
-  <arg name="classify_traffic_light_type" default="0"/>
-
-  <arg name="build_only" default="false" description="shutdown node after TensorRT engine file is built"/>
```

Comment:
[MUST] Could you please keep the build_only parameters in the launch file?
Since it is NOT a node parameter, it is used for controlling the node from outside, mainly during the pre-task in Autoware Evaluator.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/6216/commits/76e0585df26722a472f0614f8b49127527393303.